Jointify/Model/PoseMachineLearning/PoseNet.swift

```diff
@@ -155,33 +162,30 @@ class PoseNet {
         return dstImage
     }
     
-    // Calculate the angle between two joints
-    // Returns a Float degree number.
-    //
-    func calcAngleBetweenJoints() -> Float {
-        var jointNames: [JointName]
-        var jointPositions = [String: Float]()
-        var innerAngle: Float = 0.0
-        
-        switch side {
-        case .left:
-            jointNames = [.leftHip, .leftKnee, .leftAnkle]
-        case .right:
-            jointNames = [.rightHip, .rightKnee, .rightAnkle]
-        }
-        
+    // Get X and Y coordinates of joints and place them in the dictionary
+    // This function depends on pose, which only is created in the predict() function
+    // Thus, it can only be called once the prediction function is done
```

Comment:
Make this a docstring
```suggestion
    /// Get X and Y coordinates of joints and place them in the dictionary
    /// This function depends on pose, which only is created in the predict() function
    /// Thus, it can only be called once the prediction function is done
```